build_tools/kokoro/gcp_ubuntu/cmake/riscv/riscv64-linux/test.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Test the cross-compiled RISCV targets using Kokoro.
+
+set -e
+set -x
+
+# Print the UTC time when set -x is on
+export PS4='[$(date -u "+%T %Z")] '
+
+# Docker image has the QEMU installed at /usr/src/qemu-riscv.
+# Run the simple_embedding binaries under QEMU.
+ROOT_DIR=$(git rev-parse --show-toplevel)
+BUILD_HOST_DIR="${ROOT_DIR}/../build-host"
+BUILD_RISCV_DIR="${ROOT_DIR}/../build-riscv"
```

Comment:
These need to line up with existing env vars in another file. Maybe have a file external to both set them instead?

Reply:
Set the environment variables at `build_tools/kokoro/gcp_ubuntu/cmake/linux/riscv64/build.sh` so it can be shared by the two subsequent build and test scripts.